batch_seqr_loader/batch_workflow.py

```diff
@@ -121,35 +123,47 @@
     is_flag=True,
     help='Create checkpoints for intermediate Hail data',
 )
+@click.option(
+    '--skip-ped-checks',
+    'skip_ped_checks',
+    is_flag=True,
+    help='Skip checking provided sex and pedigree against the inferred one',
+)
 @click.option('--vep-block-size', 'vep_block_size')
 def main(
     gvcf_buckets: List[str],
     bam_buckets: List[str],
     bam_to_realign_buckets: List[str],
-    data_bucket: str,
+    dataset_name: str,
+    dataset_version: str,
+    output_namespace: str,
     ped_fpath: str,
-    work_bucket: str,
     keep_scratch: bool,
     overwrite: bool,
     dry_run: bool,
     disable_validation: bool,  # pylint: disable=unused-argument
     remap_path: str,
     subset_path: str,
     make_checkpoints: bool,
+    skip_ped_checks: bool,
+    namespace: Optional[str] = None,
     vep_block_size: Optional[int] = None,  # pylint: disable=unused-argument
-):
-    """
-    Entry point for a batch workflow
-    """
+):  # pylint: disable=missing-function-docstring
     if not (gvcf_buckets or bam_buckets or bam_to_realign_buckets):
         raise click.BadParameter(
             'Specify at least one of the input parameters '
             '(can be multiple and/or repeated): '
             '--gvcf-bucket, --bam-bucket, --bam-to-realign-bucket'
         )
 
+    project = 'seqr'
```

Comment:
```suggestion
    project = 'seqr-308602'
```

Reply:
It's a base for bucket names (`gs://cpg-seqr-test`), so just `seqr` :)